harpy/commands/diagnose.py

```
@@ -62,8 +73,108 @@ def diagnose(directory):
+                        cmd.append(_)
+                    continue
+            if _shellblock:
+                if line.strip() == "(command exited with non-zero exit code)":
+                    break
+                cmd.append(line.strip())
+    if not _found:
+        CONSOLE.print(f"No errors found in {os.path.basename(latest_log)}", style = "green")
+
+    if conda:
+        print_shellcmd_simple("\n".join(cmd))
+        os.system("\n".join([conda, f"cd {directory}", *cmd]))
```

Ruff (0.14.1)

Starting a process with a shell, possible injection detected

(S605)